integration/tests/tracer/test_override_params.py

```diff
@@ -0,0 +1,800 @@
+@pytest.mark.usefixtures("accounts", "web3_client", "tracer_api")
+class TestTracerOverrideParams:
+    web3_client: NeonChainWeb3Client
+    accounts: EthAccounts
+    tracer_api: TracerClient
+    storage_value = 57
+
+    @pytest.fixture(scope="class")
+    def retrieve_block_tx(self, storage_contract, web3_client):
+        sender_account = self.accounts[0]
+        nonce = web3_client.eth.get_transaction_count(sender_account.address)
+        instruction_tx = storage_contract.functions.storeBlock().build_transaction(
+            {
+                "nonce": nonce,
+                "gasPrice": web3_client.gas_price(),
+            }
+        )
+        receipt = web3_client.send_transaction(sender_account, instruction_tx)
+        assert receipt["status"] == 1
+
+        return self.web3_client.wait_get_transaction_by_hash(receipt["transactionHash"].hex())
+    
+    @pytest.fixture(scope="class")
+    def retrieve_block_timestamp_tx(self, storage_contract, web3_client):
+        sender_account = self.accounts[0]
+        nonce = web3_client.eth.get_transaction_count(sender_account.address)
+        instruction_tx = storage_contract.functions.storeBlockTimestamp().build_transaction(
+            {
+                "nonce": nonce,
+                "gasPrice": web3_client.gas_price(),
+            }
+        )
+        receipt = web3_client.send_transaction(sender_account, instruction_tx)
+        assert receipt["status"] == 1
+
+        return self.web3_client.wait_get_transaction_by_hash(receipt["transactionHash"].hex())
+
+    @pytest.fixture(scope="class")
+    def contract_index(self, call_store_value_tx):
+        index = ""
+        address_to = call_store_value_tx["to"].lower()
+        params = self.fill_params_for_storage_contract_trace_call(call_store_value_tx)
+
+        params.append({"tracer": "prestateTracer"})
+        response = self.tracer_api.send_rpc_and_wait_response("debug_traceCall", params)
+
+        assert "error" not in response, "Error in response"
+        for k, v in response["result"][address_to]["storage"].items():
+            if v == padhex(hex(self.storage_value), 64) and k != index_0:
+                index = k
+                break
+        return index
+
+    @pytest.fixture(scope="class")
+    def call_storage_tx(self, storage_contract, web3_client):
+        sender_account = self.accounts[0]
+        _, _, receipt = call_storage(sender_account, storage_contract, self.storage_value, "blockNumber", web3_client)
+        return self.web3_client.wait_get_transaction_by_hash(receipt["transactionHash"].hex())
+
+    @pytest.fixture(scope="class")
+    def call_store_value_tx(self, storage_contract, web3_client):
+        sender_account = self.accounts[0]
+        receipt = store_value(sender_account, self.storage_value, storage_contract, web3_client)
+        return self.web3_client.wait_get_transaction_by_hash(receipt["transactionHash"].hex())
```

Comment:
the store_value function include waiting for transaction, you don't need it
looks like fixture call_store_value_tx complitely the same as store_value function

Reply:
`store_value` returns receipt of tx, `call_store_value_tx` returns tx object